api/envoy/extensions/filters/http/fault/v3/fault.proto

```diff
@@ -41,6 +42,9 @@ message FaultAbort {
     // HTTP status code to use to abort the HTTP request.
     uint32 http_status = 2 [(validate.rules).uint32 = {lt: 600 gte: 200}];
 
+    // gRPC status code to use to abort the gRPC request.
+    uint32 grpc_status = 5 [(validate.rules).uint32 = {lte: 16}];
```

Comment:
@tonya11en do you recall where we ended up with gRPC as int vs. enum?

Reply:
We landed on using integers, so I scrapped the enum.